src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-sidebar/dot-edit-layout-sidebar.component.ts

```diff
@@ -74,7 +76,7 @@ export class DotEditLayoutSidebarComponent implements OnInit, ControlValueAccess
      * @param {DotContainerColumnBox[]} containers
      * @memberof DotEditLayoutSidebarComponent
      */
-    updateContainers(containers: DotContainerColumnBox[]): void {
+    updateContainers(containers?: DotContainerColumnBox[]): void {
         this.propagateChange(this.getModel(containers));
     }
 
```

Comment:
any test is updated with this change?

Reply:
you're right, adding 2 more test for that!